content/en/docs/setup/platform-setup/kubesphere/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: KubeSphere Container Platform
+description: Instructions to setup a KubeSphere Container Platform for Istio.
+weight: 19
+skip_seealso: true
+keywords: [platform-setup,kubesphere,kubernetes]
+---
+
+Follow these instructions to prepare the [KubeSphere Container Platform](https://github.com/kubesphere/kubesphere) for Istio. You can download KubeSphere to easily install a Kubernetes cluster on your Linux machines.
+
+{{< tip >}}
+KubeSphere provides [All-in-One](https://kubesphere.io/docs/v2.1/en/installation/all-in-one/) and [Multi-Node](https://kubesphere.io/docs/v2.1/en/installation/multi-node/) installations. This enables quick setup and manages Kubernetes and Istio in a unified web console. This tutorial will walk you through the All-in-One installation. Reference [Multi-node Installation](https://kubesphere.io/docs/v2.1/en/installation/multi-node/) for further information.
+{{< /tip >}}
+
+## Prerequisites
+
+A new Linux machine, supports virtual machine or bare metal, the minimum requirements are listed:
+
+- Hardware
+   - CPU：at least 2 Cores
```

Comment:
The `isto_lint` check shows line 20 has [unordered list indentation error](https://prow.istio.io/view/gcs/istio-prow/pr-logs/pull/istio_istio.io/6424/lint_istio.io/4314), but I verify the corrected example from [MD007 - Unordered list indentation](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md007---unordered-list-indentation) is the same with here, I am not sure if there is any mistake in line 20. Can anybody help to check?  

Thanks